### PR TITLE
Awesome books review

### PR DESCRIPTION
**Pull request by akezeth**
- Implemented basic UI with plain HTML.
- Created a collection using a linked list that keeps a list of books.
- Created a function to add a new book to the collection, with title and author.
- Created a function to remove a book from the collection.
- Used localStorage to preserve data in the browser's memory.

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi team @akezeth, @chawkimbarki 🙋‍♀️,  

While you made a great effort in this project, I can't review your code.

## Invalid Code Review Request
 
You have submitted a project that does not have a documented README. Kindly document your README using the new Microverse [README template](https://github.com/microverseinc/readme-template).
Here are key points to consider when documenting your README to make it professional.

- The template logo should be removed. Optionally, the Microverse logo can be replaced with your own project logo if you have one. ✖️ 
- The only optional section is the FAQ section. If there's no information in the FAQ section then it should be deleted. ✖️ 
***If you decide to delete the FAQ section, it would be best if you also removed it from the table of contents.***
- All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present. Even if there's no information for a section or sub-section, rather leave texts like "NA", "None" than delete it except for the Live Demo section. ✖️ 
- Future Features section should include 1-3 potential features to be added. ✖️ 
-  Each section description should be removed and replaced with their custom content. The section descriptions are meant to guide you in documenting your README. They are not meant to be part of the final documentation. ✖️ 

For example:
image
image
image

- The License suggestion should be removed. ✖️ 
image

- There should not be any blank sections that do not have content. ✖️ 

Kindly document the `Show your support` section.
image
Kindly document the `Acknowledgments` section.
image


- There should not be any broken links in the README file. ✖️ 
- Delete the live demo section if the application is not deployed. If project requirements say to deploy the project, the section should be restored and the link updated. ✖️ 

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by akezeth** [comment]
Updated README.md file

File: script.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+const books = grabbingBooks();
+const btn = document.getElementById('add');
+
+btn.addEventListener('click', () => {
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  const newBook = new Book(title, author);
+  books.add(newBook);
+  displayBook(newBook, books.size - 1);
+});
```
Comment (AmaduKamara) [inline script.js:138]: - [ ] I tried adding a book but nothing happens. Would you please work on adding a book to an array of objects as required in the project requirements? 👍🏽
- [ ] It's required that when the add button is clicked, a book is added to the books array and is again saved to localStorage and then fetched and rendered on the page. All that is not working as expected. Please work on that 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi Team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly work on the app to be able to add a book when the form is filled and the add button is clicked. A new book is expected to be added to an array of objects, stored in localStorage, and then fetched and rendered on the page.
- [ ] Please be sure that the removal of a book works fine as well. At the moment that is not testable because a book cant's be added 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by akezeth** [comment]
Adjusted the add and remove errors.

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @TEAM     , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯 
- [x] linters are passing
- [x] Professional README 💯
- [x] Awesome books app running as intended  🥇  
 










 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]

**Pull request merged**